nx/lib/nx/random.ex

```diff
@@ -44,6 +44,26 @@ defmodule Nx.Random do
      (generating array values) and multi-replica, multi-core
      computation. In particular it should not use sequencing
      constraints between random function calls.
+
+  <a name="key-to-keys"></a>
```

Comment:
I took the idea from the Elixir docs :(
I wanted to add this because it's something that people are bound to ignore if it's not written In flashing colors

Reply:
So it is either the admonition block or the link I think. Sorry :(

Reply:
Oh, I though you were referring to the block itself. I'll remove the link as soon as I can :)